examples/common/builders.py

```diff
@@ -27,6 +28,8 @@ def build_callback(name, kwargs):
         return SpeedMonitorMFU(window_size=kwargs.get('window_size', 1),
                                gpu_flops_available=kwargs.get(
                                    'gpu_flops_available', None))
+    elif name == 'fdiff_loss':
+        return FDiffLoss()
```

Comment:
Since this is diff-ing all metrics, why not just `fdiff` and `FDiffMetrics` or something like that?